src/dotnet-core-uninstall/Shared/Configs/OptionFilterers.cs

```diff
@@ -1,44 +1,43 @@
 ﻿using System.Collections.Generic;
-using System.CommandLine;
 using Microsoft.DotNet.Tools.Uninstall.Shared.Filterers;
 
 namespace Microsoft.DotNet.Tools.Uninstall.Shared.Configs
 {
     internal static class OptionFilterers
     {
-        public static readonly IDictionary<Option, Filterer> OptionFiltererDictionary
-            = new Dictionary<Option, Filterer>
+        public static readonly IDictionary<string, Filterer> OptionFiltererDictionary
```

Comment:
Do we have to use string here? Option is a stronger type and can have more type safety (and also better "documentation" considering it is clear only a certain type can put in the dictionary instead of any string)